docs/exceptions.rst

```diff
@@ -0,0 +1,56 @@
+Runtime Exceptions
+==================
+
+
+Runtime exceptions can be thrown by the graphql engine if certain exceptional situations are encountered.  The following
+are a list of the exceptions that can be thrown all the way out of a ``graphql.execute(...)`` call.
+
+These are not graphql errors in execution but rather totally unacceptable conditions in which to execute a graphql query.
+ 
+ -  `graphql.schema.CoercingSerializeException`
```

Comment:
I don't think this exception can escalate to the end user a exception thrown from `.execute` ... we handle it in `handleCoercionProblem`